ci-operator/step-registry/image-based-install-operator/post/delete-ici/image-based-install-operator-post-delete-ici-commands.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ image based install operator post delete ici command ************"
+
+source "${SHARED_DIR}/packet-conf.sh"
+
+ssh "${SSHOPTS[@]}" "root@${IP}" bash - << "EOF"
+
+# prepending each printed line with a timestamp
+exec > >(awk '{ print strftime("[%Y-%m-%d %H:%M:%S]"), $0 }') 2>&1
+
+set -xeo pipefail
+
+source /root/env.sh
+
+cd /root/dev-scripts
+source common.sh
+source utils.sh
+source network.sh
+
+echo "### Delete the ImageClusterInstall"
+oc delete imageclusterinstall ibi-cluster -n ibi-cluster
+oc wait -n ibi-cluster --for=delete imageclusterinstalls.extensions.hive.openshift.io/ibi-cluster --timeout=10m
+oc wait -n openshift-machine-api --for=delete dataimages.metal3.io/ostest-extraworker-0 --timeout=60m
```

Comment:
60 minutes seems too long, if it doesn't get deleted after 10 it won't get deleted after 60.

Reply:
From the logs I saw it takes exactly 10 minutes, so maybe 20 minutes timeout?